app/models/work.rb

```diff
@@ -321,9 +323,8 @@ def self.successful_reindex(ids)
     Collection.expire_ids(collection_ids)
   end
 
-  # TODO: AO3-6085 We can use touch_all once we update to Rails 6.
   def touch_series
```

Comment:
Should this be added to the `after_destroy` hooks in addition to `expire_caches`? Or maybe `expire_caches` could call `touch_series`?

Reply:
I apologise if I'm understanding you incorrectly -- but I believe I added a call to expire series caches in `expire_caches` [here](https://github.com/otwcode/otwarchive/pull/4558/files#diff-ebe838f9d71d26b6227d6d37c96a7e2d93fb68a80d55075c7f2374bfcb4688abR262), which to my understanding touches series.

Reply:
I don't think it would touch the `Series` itself, just the works associated with it. I did find https://github.com/otwcode/otwarchive/pull/3976/commits/409157dfb13edf4094c62af3c47f08c15254380a#r682235987 though, which suggests this was intentional. So feel free to resolve this thread!